src/web/src/components/Posts/Post.tsx

```diff
@@ -152,42 +205,52 @@ const PostComponent = ({ postUrl }: Props) => {
   }
 
   return (
-    <Box className={classes.root} boxShadow={2}>
-      <ListSubheader className={classes.header}>
-        <AdminButtons />
-        <Typography variant="h1" title={post.title} id={post.id} className={classes.title}>
-          <span
-            role="button"
-            tabIndex={0}
-            onClick={() => setExpandHeader(!expandHeader)}
-            onKeyDown={() => setExpandHeader(!expandHeader)}
-            className={expandHeader ? classes.expandHeader : classes.collapseHeader}
-          >
-            {post.title}
-          </span>
-        </Typography>
-        <Typography variant="caption" className={classes.author}>
-          &nbsp;By&nbsp;
-          <a className={classes.link} href={post.feed.link}>
-            {post.feed.author}
-          </a>
-        </Typography>
-        <a href={post.url} rel="bookmark" className={classes.published}>
-          <time className={classes.time} dateTime={post.updated}>
-            {` ${formatPublishedDate(post.updated)}`}
-          </time>
-        </a>
-      </ListSubheader>
+    <Box className={classes.root}>
+      <Container className={classes.titleAndAuthorContainer}>
+        <ListSubheader className={classes.titleContainer}>
+          <AdminButtons />
+          <Typography variant="h1" title={post.title} id={post.id} className={classes.title}>
+            <span
+              role="button"
+              tabIndex={0}
+              onClick={() => setExpandHeader(!expandHeader)}
+              onKeyDown={() => setExpandHeader(!expandHeader)}
+              className={expandHeader ? classes.expandHeader : classes.collapseHeader}
+            >
+              {post.title}
+            </span>
+          </Typography>
+        </ListSubheader>
+        <ListSubheader className={classes.postInfo}>
+          <div className={classes.authorInfoContainer}>
+            <div className={classes.authorAvatarContainer}>
```

Comment:
Can we switch this to https://material-ui.com/components/avatars/#avatar so that it will also work when we have an image?

Reply:
Filed #1855 to deal with this.

Reply:
> This is really cool to see happening. I left a few comments in the code, but here are some visual things I notice:
> 
> I think the font size on the title is too large, and it cuts off most typical length titles:
> 
> <img alt="Screen Shot 2021-02-28 at 6 29 45 PM" width="1021" src="https://user-images.githubusercontent.com/427398/109437496-069e8d00-79f3-11eb-874d-b6372fc8bd50.png">
> 
> Line-height on wrapped names is too much (should be single I think?)
> 
> <img alt="Screen Shot 2021-02-28 at 6 30 55 PM" width="223" src="https://user-images.githubusercontent.com/427398/109437515-25048880-79f3-11eb-9af7-22fccd84fe53.png">
> 
> The name and date shouldn't carry the same "weight":
> 
> <img alt="Screen Shot 2021-02-28 at 6 31 46 PM" width="167" src="https://user-images.githubusercontent.com/427398/109437544-48c7ce80-79f3-11eb-9eed-7a098168b5d0.png">
> 
> I think the date could be a lighter shade, and maybe smaller. It's important, but not as important as the name. And the name is probably not as important as the text of the post. Consider how Medium does this:
> 
> ![Screen Shot 2021-02-28 at 6 33 16 PM](https://user-images.githubusercontent.com/427398/109437573-7745a980-79f3-11eb-852f-c166e81a21dd.png)

@humphd I totally agree with you. I make the changes. When we fix the font inconsistency problem (#1856) we may need to revisit this part.

Reply:
Can you file an issue on these specific issues though, so we don't lose track?

Reply:
@humphd I already filed the issues.

Reply:
Awesome, thank you!